### ISSUE TITLE
What do you think about porting CocosBuilder to Qt for real cross platform ?

### ISSUE THREAD

**Issue by meiry** [cocos2d/CocosBuilder#432]
since this project is dead . 
i was thinking about idea to port this project to Qt for other users also 
its great app is a shame to let it die .

**Comment by JogsWu** [comment]
Yes, vlidholt said that he didn't work for CocosBuilder, maybe we can work together to port it to Qt.

**Comment by edudemy** [comment]
+1
To cross port this project to Qt _and_ include updated CCBReaders to the C++ Cocos2d-x project.
I guess the bigger challenge is to keep the CCBReader updated and in sync with the API changes.

**Comment by meiry** [comment]
well asked 3 moths ago and there is poor response , i guess there is no demand for it , 
i wander if cocos-studio will be good replacement

**Comment by edudemy** [comment]
There is no CocoStudio for Mac yet. Apportable/Spritebuilder is the only cross-platform(iOS/android) solution for Mac users.

If  CocosBuilder is updated to work with the latest cocos2d-x 3.0 release then a lot of Mac developers will get truly cross-platform(c++/js) alternative to SpriteBuilder without being restricted to Obj-C/Apportable.

However, Viktor's current employment with Apportable prevents his earlier roadmap vision from being pursued due to conflict of interest with Apportable.

Cocosbuilder 3.0 C++ Branch would find lots of takers looking for cross-platform development
https://github.com/cocos2d/CocosBuilder/blob/master/Documentation/XX1.%20CocosBuilder%203%20Roadmap.md

**Comment by AlexanderBykin** [comment]
rewrite it with Java + JavaFX then it will be trully crossplatform

**Comment by edudemy** [comment]
Can somebody pls upload a working binary of CocosBuilder 3?  I tried building the latest project here on github with Xcode. But It failed.

**Comment by meiry** [comment]
well one thing i do in such tool it replacing the ccb files with real code generation option 
so the class will be compiled and not load

**Comment by edudemy** [comment]
:+1:
